packages/web/src/components/tracks-table/TracksTable.js

```diff
@@ -39,25 +40,28 @@ export const alphaSortFn = function (a, b, aKey, bKey) {
   return a.toLowerCase() > b.toLowerCase() ? 1 : -1
 }
 
+const allRowsActionButtonRefs = []
```

Comment:
how often do we call favoriteButtonCell? if it's part of a render cycle, couldn't allRowActionButtonRefs grow infinitely?

Reply:
good question. I'll see if it grows over multiple renders

Reply:
yeah this is a problem. It adds them on every render. Gonna try to make it key off trackid or something

Reply:
fixed on the latest which keys the stored refs by the table row's key